doc/run_experiment.rst

```diff
@@ -921,121 +949,169 @@ hyperparameters. Available metrics are:
     *   **uwk_off_by_one**: Same as ``unweighted_kappa``, but all ranking
         differences are discounted by one. In other words, a ranking of
         1 and a ranking of 2 would be considered equal.
-    
-
-.. _param_grids:
-
-param_grids *(Optional)*
-""""""""""""""""""""""""
-
-List of parameter grids to search for each learner. Each parameter
-grid should be a list of dictionaries mapping from strings to lists
-of parameter values. When you specify an empty list for a learner,
-the default parameter grid for that learner will be searched.
-
-The default parameter grids for each learner are:
-
-AdaBoostClassifier and AdaBoostRegressor
-    .. code-block:: python
-
-        [{'learning_rate': [0.01, 0.1, 1.0, 10.0, 100.0]}]
-
-BayesianRidge
-    .. code-block:: python
 
-        [{'alpha_1': [1e-6, 1e-4, 1e-2, 1, 10],
-          'alpha_2': [1e-6, 1e-4, 1e-2, 1, 10],
-          'lambda_1': [1e-6, 1e-4, 1e-2, 1, 10],
-          'lambda_2': [1e-6, 1e-4, 1e-2, 1, 10]}]
+The following is a list of the other optional fields in this section in alphabetical order.
 
-DecisionTreeClassifier and DecisionTreeRegressor
-    .. code-block:: python
-
-       [{'max_features': ["auto", None]}]
-
-ElasticNet
-    .. code-block:: python
-
-       [{'alpha': [0.01, 0.1, 1.0, 10.0, 100.0]}]
-
-GradientBoostingClassifier and GradientBoostingRegressor
-    .. code-block:: python
-
-       [{'max_depth': [1, 3, 5]}]
-
-HuberRegressor
-    .. code-block:: python
-
-        [{'epsilon': [1.05, 1.35, 1.5, 2.0, 2.5, 5.0],
-          'alpha': [1e-4, 1e-3, 1e-3, 1e-1, 1, 10, 100, 1000]}]
-
-KNeighborsClassifier and KNeighborsRegressor
-    .. code-block:: python
-
-        [{'n_neighbors': [1, 5, 10, 100],
-          'weights': ['uniform', 'distance']}]
-
-Lasso
-    .. code-block:: python
-
-       [{'alpha': [0.01, 0.1, 1.0, 10.0, 100.0]}]
+.. _grid_search:
 
-LinearSVC
-    .. code-block:: python
+grid_search *(Optional)*
+""""""""""""""""""""""""
 
-       [{'C': [0.01, 0.1, 1.0, 10.0, 100.0]}]
+Whether or not to perform grid search to find optimal parameters for
+classifier. Defaults to ``True`` since optimizing model hyperparameters
```

Comment:
We should probably change "classifier" to something more general, like "learner"